syncer/relay.go

```diff
@@ -120,5 +120,5 @@ func (s *Syncer) removeActiveRelayLog() {
 	}
 
 	s.readerHub.RemoveActiveRelayLog(s.cfg.Name)
-	log.Infof("[syncer] current earliest active relay log %s", s.readerHub.EarliestActiveRelayLog())
+	s.tctx.L().Info("current earliest active relay log", zap.Stringer("active relay log", s.readerHub.EarliestActiveRelayLog()))
```

Comment:
Currently `s.readerHub.EarliestActiveRelayLog()` may be nil, seems unexpected, we should always get an avialable EarliestActiveRelayLog after sync unit starts?

Reply:
yeh, we should call `setInitActiveRelayLog` in `Syncer.Init`.

Reply:
anyway I use `WrapStingerField`